goodtablesio/blueprints/site.py

```diff
@@ -9,18 +10,29 @@
 
 @site.route('/')
 def home():
-    return render_template('home.html')
+    user_jobs = []
+    user_id = session.get('user_id')
+    if user_id:
+        # TODO: limit jobs count to show!
+        # TODO: here we should filter jobs by user!
```

Comment:
I guess we need to think about what do we want to show users by default. My feeling is that we don't want to have repos (or jobs run on them) linked to a particular user. Say that you activate the hook on a particular repo on the frictionlessdata organization on GH. Should I see the jobs run on that repo? I would say yes, but if we use individual users to decided this it gets very complicated (I would need to "Follow" somehow that repo). 
A better approach is probably to display all jobs from the organizations that you belong to. This might have implications with private repos later on but for public ones I think it's the easiest.